platform/darwin/src/MGLFillStyleLayer.h

```diff
@@ -96,6 +96,9 @@ typedef NS_ENUM(NSUInteger, MGLFillTranslateAnchor) {
  */
 @property (nonatomic, null_resettable) MGLStyleValue<NSString *> *fillPattern;
 
+
+- (void)addToMapView:(MGLMapView *)mapView;
```

Comment:
This method needs to be private. We want developers to continue to call `-[MGLStyle addLayer:]` or `-[MGLStyle insertLayer:belowLayer:]` as appropriate. To avoid having to create a separate private header for each layer subclass, declare it in the private header for MGLStyle.